tests/autorestart_test/test_container_autorestart.py

```diff
@@ -213,8 +213,9 @@ def test_containers_autorestart(duthost):
     disabled_containers = get_disabled_container_list(duthost)
 
     for container_name in container_autorestart_states.keys():
-        # Skip testing the database container or containers/services which are disabled
-        if container_name in disabled_containers or container_name == "database":
+        # Skip testing the database container, radv container on T1 devices and containers/services which are disabled
+        if (container_name in disabled_containers or container_name == "database"
+                or (container_name == "radv" and tbinfo["topo"]["type"] != "t1")):
```

Comment:
According to https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-router-advertiser/docker-router-advertiser.supervisord.conf.j2#L30, radvd is only available in ToR(T0) devices. Should the logic here be like below?

```
	        if (container_name in disabled_containers or container_name == "database"
                    or (container_name == "radv" and tbinfo["topo"]["type"] != "t0")):
```

Reply:
Yes, Xin, your suggestion is correct. Thanks.

@yozhao101: Please adjust as per Xin's suggestion

Reply:
Good catch! Thanks.